system/ee/ExpressionEngine/Addons/channel/libraries/channel_form/Channel_form_lib.php

```diff
@@ -1642,7 +1642,7 @@ function ($v) {
         }
 
         if ($this->json) {
-            return $this->send_ajax_response(
+            return ee()->output->send_ajax_response(
```

Comment:
Since we're changing the call here should we deprecate or remove the `Channel_form_lib@send_ajax_response()` method? https://github.com/ExpressionEngine/ExpressionEngine/blob/d9157e68ccf7ff8c11820f2b94e1481bb050c1b1/system/ee/ExpressionEngine/Addons/channel/libraries/channel_form/Channel_form_lib.php#L2920-L2938 It looks like a duplicate of what the Output class is doing, but it is public and possibly used by third parties.  Should we at least bring it inline with the Output class and remove the user agent detection and conditional?

Reply:
It might be good to deprecate it, but `ee()->output->send_ajax_response` is also using legacy `Output` class which will might be replaced with something else in EE7. I'm not sure how good it to deprecate in favor of something that we expect to eventually be gone as well. If anyone uses this function, they would still see deprecation message related to `user_agent`